chess-AI/train.py

```diff
@@ -1,32 +1,46 @@
-from functools import reduce
-
-import chess
+"""
+Main training program
+"""
+import time
 import os
+import chess
 import torch
-import numpy as np
+
 from torch.utils.data import DataLoader, TensorDataset
 
 from mcts import Mcts
 from ai_io import save_model, load_model
 from nn_layout import PlayNetwork
 from output_representation import PlayNetworkPolicyConverter
 from state_representation import get_cnn_input
-
+from stockfish_train import StockfishTrain
+from streamlit_dashboard import StreamlitDashboard
 
 class Train:
-    """This class is used to run the monte carlo simulations and drive the model training.
+    """The main training class, will train using stockfish and mcts
+    
+    1. Will create a dataset or load a dataset of (fen_strings,move_probs,values)
```

Comment:
You say create or load; when will a dataset be loaded? If it is an optional thing, maybe we need to add some parameter to the constructor to load the data.